.github/workflows/build.yaml

```diff
@@ -26,8 +26,8 @@ on:
         required: false
         default: ''
       build_from_cache:
-        description: 'Whether to build images from cache or not. Defaults to true, because not doing so significantly increases build time'
-        required: false
+        description: 'Whether to build images from cache or not. Default: true, set to false only if required because that will cause significantly increase in build time'
```

Comment:
```suggestion
        description: 'Whether to build images from cache or not. Default: true, set to false only if required because that will cause a significant increase in build time'
```